ingester/ingester.go

```diff
@@ -38,6 +38,8 @@ const (
 	DefaultConcurrentFlush = 50
 	// DefaultMaxSeriesPerUser is the maximum number of series allowed per user.
 	DefaultMaxSeriesPerUser = 5000000
+	// DefaultMaxSeriesPerMetric is the maximum number of series in one metric (of a single user).
+	DefaultMaxSeriesPerMetric = 50000
```

Comment:
We need to do some testing to figure out this limit; previously 20k was too many, but it should be more now.  Can you file a follow up issue please?